client/coral-admin/src/containers/Community/Table.js

```diff
@@ -1,34 +1,66 @@
-import React from 'react';
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {SelectField, Option} from 'react-mdl-selectfield';
 import styles from './Community.css';
+import I18n from 'coral-framework/i18n/i18n';
+import translations from '../../translations';
+import {setRole} from '../../actions/community';
 
-const Table = ({headers, data, onHeaderClickHandler}) => (
-  <table className={`mdl-data-table ${styles.dataTable}`}>
-    <thead>
-      <tr>
-        {headers.map((header, i) =>(
-          <th
-          key={i}
-          className="mdl-data-table__cell--non-numeric"
-          onClick={() => onHeaderClickHandler({field: header.field})}>
-            {header.title}
-          </th>
-        ))}
-      </tr>
-    </thead>
-    <tbody>
-      {data.map((row, i)=> (
-        <tr key={i}>
-          <td className="mdl-data-table__cell--non-numeric">
-            {row.displayName}
-            <span className={styles.email}>{row.profiles.map(({id}) => id)}</span>
-          </td>
-          <td className="mdl-data-table__cell--non-numeric">
-            {row.created_at}
-          </td>
-        </tr>
-      ))}
-    </tbody>
-  </table>
-);
+const lang = new I18n(translations);
 
-export default Table;
+class Table extends Component {
+
+  constructor (props) {
+    super(props);
+    this.onRoleChange = this.onRoleChange.bind(this);
+  }
```

Comment:
All the functionality of Community is in the container `/containers/Community/CommunityContainer.js`. Can we keep this one stateless? It's not mandatory. So we avoid connecting and classing this one too. (I don't mind doing it)